core/src/main/scala/kafka/server/ControllerServer.scala

```diff
@@ -467,7 +470,7 @@ class ControllerServer(
         CoreUtils.swallow(quotaManagers.shutdown(), this)
       Utils.closeQuietly(controller, "controller")
       Utils.closeQuietly(quorumControllerMetrics, "quorum controller metrics")
-      authorizer.foreach(Utils.closeQuietly(_, "authorizer"))
+      authorizerPlugin.foreach(Utils.closeQuietly(_, "authorizer"))
```

Comment:
ditto: Should we rename "authorizer" to "authorizer plugin" for clarity? Since we explicitly close the authorizer in `Plugin#close()`.